example/package.json

```diff
@@ -4,10 +4,10 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ../node_modules/react-scripts/bin/react-scripts.js start",
-    "build": "node ../node_modules/react-scripts/bin/react-scripts.js build",
-    "test": "node ../node_modules/react-scripts/bin/react-scripts.js test",
-    "eject": "node ../node_modules/react-scripts/bin/react-scripts.js eject"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
with your modification to the package.json of the example, I can't start it anymore: 
![image](https://user-images.githubusercontent.com/9303791/197052157-fd1ebf0a-880c-48f6-993a-2b43b6f9d982.png)

Reply:
reverted my change, should be ok now.

Reply:
weird